index.rst

```diff
@@ -10,35 +10,39 @@ The aim of this page is to clearly describe all the activities and materials so
 
 .. contents:: Table of Content
 
-Workshops and tutorials
------------------------
+Workshops and tutorials organised by ECE
+----------------------------------------
```

Comment:
Ah nice idea @danielhuppmann :)
I guess I would just keep it at "Workshops and tutorials" without "organised by ECE" since it is the ECE teaching website.

Reply:
The idea was to more visibly distinguish between "items organized by ECE" vs. "items at external institutions"...